docs/configuration/index.md

```diff
@@ -1525,6 +1525,19 @@ The amount of direct memory needed by Druid is at least
 ensure at least this amount of direct memory is available by providing `-XX:MaxDirectMemorySize=<VALUE>` at the command
 line.
 
+#### Indexer Security Configuration
```

Comment:
I find this section confusing because the `Indexer` here means the node type `Indexer not ingestion. Can you move the new docs to [Ingestion Security Configuration](https://github.com/apache/druid/blob/master/docs/configuration/index.md#ingestion-security-configuration)?